tests/runtest.proj

```diff
@@ -379,6 +379,26 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     <MSBuild Projects="$(MSBuildProjectFile)" Targets="UploadPerfAssemblies" />  
   </Target>  
 
+  <PropertyGroup>
+    <CMakeCache>$(RootBinDir)obj\$(BuildOS).$(BuildArch).$(BuildType)\CMakeCache.txt</CMakeCache>
+    <CMakeCacheContent>$([System.IO.File]::ReadAllText($(CMakeCache)))</CMakeCacheContent>
```

Comment:
It's a nice feature to be able to say "don't fail the test because I forced PGO off", but I'm wondering if we really want this.  What if PGO gets forced off in the build lab by accident or in Jenkins runs?  We'd rather get a test failure and know about it.  I think it may be best to unconditionally run the test.